src/main/java/com/gluonhq/substrate/util/FileDeps.java

```diff
@@ -75,7 +75,7 @@
 
     private static final String ANDROID_KEY = "24333f8a63b6825ea9c5514f83c2829b004d1fee";
     private static final String[] ANDROID_SDK_PACKAGES = {
-            "platforms;android-27", "build-tools;27.0.3", "platform-tools", 
+            "platforms;android-27", "build-tools;29.0.2", "platform-tools",
```

Comment:
I get this warning at the end of the package task:
```
WARNING: No version of NDK matched the required version 21.1.6352462. Versions available locally: 
```
I have installed the ndk-bundle, but somehow it doesn't find it. Not sure if this is an issue?

Reply:
It is known issue at Google's end https://issuetracker.google.com/issues/144111441#comment16
If package task doesn't fail I think we can safely ignore it.

Reply:
Also, Gradle doesn't actually use NDK in this project configuration, since we are linking everything in `AbstractTarget` and just copying result as shared library.